spark-kafka-0-10-writer/src/main/scala/com/github/benfradet/spark/kafka010/writer/DStreamKafkaWriter.scala

```diff
@@ -41,10 +41,11 @@ class DStreamKafkaWriter[T: ClassTag](@transient private val dStream: DStream[T]
    */
   override def writeToKafka[K, V](
     producerConfig: Properties,
-    transformFunc: T => ProducerRecord[K, V]
+    transformFunc: T => ProducerRecord[K, V],
+    callback: Option[Callback]
```

Comment:
Will do, had already set the default in the super class to avoid compatibility issues but can add it in the implementing classes too.